app/views/shared/_nav.html.slim

```diff
@@ -15,11 +15,12 @@ nav.bg-white
         - if signed_in?
           .h4 Welcome <strong>#{current_user.email}</strong>
           .mt1.sm-mt2.h5
-            = link_to t('links.my_account'), profile_path, \
-              class: 'bold underline'
+            - if current_page?(profile_path)
+              span.bold = t('links.my_account')
```

Comment:
i feel like it should always be a link (for consistency), but when you're on the profile page, the link is bold / diff color; something like `class: "#{current_page?(profile_path) ? 'bold black' : 'underline'"`?